settings.gradle.kts

```diff
@@ -1,12 +1,20 @@
 rootProject.name = "kotlin-wrappers"
 
 pluginManagement {
+    repositories {
```

Comment:
Looks like local development settings

Reply:
I think it is ok, because it is just a prioritized list. I don't know why it is not a default behavior though.

Reply:
It was removed after discussion